Unity-Technologies/ScriptableRenderPipeline
# Add HDRenderPipeline tag to DXR subshaders

### Purpose of this PR
Fix for https://fogbugz.unity3d.com/f/cases/1184503/
Specify a render pipe for the DXR subshaders so that they aren't set active before the render pipeline is active.

---
### Testing status

**Manual Tests**: What did you do?
- Other: 
Tested the test project provided in the linked fogbugz and found that the discrepancy in pass names no longer occurs.
Opened the DXRSmallRoom project to confirm in Playmode that ray tracing still works as expected.

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)
None

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

Any test projects to go with this to help reviewers?
https://bugs.unity3d.com/fogfiles/118xxxx/1184503_fshg493k7s7n82as/case_1184503-TestFindPass.zip

---
### Comments to reviewers
Notes for the reviewers you have assigned.
